test/GENargumentTests.html

```diff
@@ -0,0 +1,119 @@
+<html><head> 
+    <title>SPOT Automated Testing</title>
+
+	<link rel="stylesheet" type="text/css" href="../spot.css">
+
+    <!--If running tests that do not have to do with the interface, run jsbuild.sh or jsbuild.bat
+        in root directory of the project. If running interface testing, run testbuild.sh
+        or testBuild.bat-->
+    <script src="../build/spot.js"></script>
+
+    <!--Mocha setup stuff (1/2)-->
+    <meta charset="UTF-8">
+    <script src="../lib/test/mocha.min.js"></script>
+    <script src="../lib/test/chai.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="../lib/test/mocha.css">
+    <!--mocha setup stuff to be continued in document body-->
+    <style type = "text/css">
+    <!--
+        .tab { margin-left: 40px; }
+        .twotab { margin-left: 80px; }
+    -->
+    </style>
+</head>
+<body style="padding-left: 5%; padding-right: 5%; padding-top: 20px">
+
+    <h2>Automated GEN Testing</h2>
+    
+    <script src="./GENoutputTrees.js"></script>
+
+    <!--Mocha stuff (2/2)-->
+    <script>
+        mocha.setup("bdd"); //brings "describe", "it", etc. into global namespace
+        //mocha.checkLeaks();
+        var assert = chai.assert; //no one wants to type out "chai.assert" every time
+    </script>
+    <div id="mocha">
+        Automated GEN Testing
+    </div>
+    
+    <!--Mocha is set up now, you just need to write and run tests-->
+    <script>
+        
+        stree = {
+            "id": "CP1",
+            "cat": "cp",
+            "children": [
+                {
+                    "id": "a",
+                    "cat": "x0"
+                },
+                {
+                    "id": "b",
+                    "cat": "x0"
+                },
+                {
+                    "id": "c",
+                    "cat": "x0"
+                }
+            ]
+        }
+
+        describe('GEN with various inputs', function() {
+            it('GEN with stree only', function() {
```

Comment:
Looks to me like you didn't finish updating this test file. The actual tests don't match their descriptions, and none of them are passing at the moment. Also I don't see why we should expect a length of 24 for any of these, since you haven't specified {rootCat:'phi'} or {wrapInRecCat:true}.